thrall/app/lib/ThrallConfig.scala

```diff
@@ -28,6 +28,11 @@ class ThrallConfig(override val configuration: Configuration) extends CommonConf
         "App"   -> Seq(elasticsearchApp)
       ))
 
+
+  lazy val elasticsearch6Host: String = properties.getOrElse("es6.host", "localhost")     // TODO how to discover the 6 cluster in AWS
+  lazy val elasticsearch6Shards: Int = if (isDev) 1 else properties.getOrElse("es6.shards", "1").toInt
```

Comment:
Maybe best not to have these default in PROD but fail if there's no config?

Reply:
Yep; sounds sensible.